pages/news.js

```diff
@@ -1,7 +1,253 @@
-import Layout from '../components/Layout.js'
+import Layout from '../components/Layout.js';
+
+const img = {
```

Comment:
Do we need this const?

Reply:
```suggestion
const img = {
```
@shyaman No it is not necessary. remove that.